src/include/optimizer/group.h

```diff
@@ -92,6 +92,9 @@ class Group {
 
   inline GroupID GetID() { return id_; }
 
+  const std::string GetInfo(int num_indent);
```

Comment:
explicitly inherit from peloton::Printable if you intend this to be usable as one